examples/kubernetes/main.go

```diff
@@ -17,7 +17,7 @@ import (
 const (
 	// berglasContainer is the default berglas container from which to pull the
 	// berglas binary.
-	berglasContainer = "us-docker.pkg.dev/berglas/berglas/berglas:latest"
+	berglasContainer = "gcr.io/berglas/berglas:latest"
```

Comment:
I'm fine with the other changes, but the gcr.io URLs are behind and will not receive future updates. We're using Artifact Registry for berglas moving forward.

Reply:
Alright, but pulling from those images doesn't work:

https://github.com/GoogleCloudPlatform/berglas/issues/128

Reply:
Okay. It works now. It didn't work for the whole day yesterday, hence the changes :)

Reply:
Restored it back to the correct repo.